cs/azure/Trainer/Operations/InitializeOperation.cs

```diff
@@ -75,7 +75,11 @@ internal void FreshStart(OnlineTrainerState state = null)
             // start from scratch
             this.state = state;
 
-            this.InitializeVowpalWabbit(new VowpalWabbitSettings(this.settings.Metadata.TrainArguments));
+            var settings = model == null ?
+                new VowpalWabbitSettings(this.settings.Metadata.TrainArguments) :
+                new VowpalWabbitSettings(string.Empty) { ModelStream = new MemoryStream(model) };
```

Comment:
This is probably wrapped in a try-catch under the PUT method right? Also, we'll likely see many user errors w.r.t. using warm start model so perhaps in the logged message, we could have a link to some wiki page for diagnosing it? By wiki page I'm thinking of general guidelines on what kind of models would be compatible etc...